cmd/clean/clean.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/buildsafe/bsf/cmd/init"
+	"os"
+)
+
+func init() {
+	rootCmd.AddCommand(cleanCmd)
+}
+
+var cleanCmd = &cobra.Command{
+	Use:   "clean",
+	Short: "Clean autogenerated BSF files and directories",
+	Run: func(cmd *cobra.Command, args []string) {
+		if os.Getenv("BSF_DEBUG_MODE") == "" {
+			fmt.Println("BSF_DEBUG_MODE is not enabled. Skipping clean.")
+			return
+		}
```

Comment:
We don't really need this as the command will only be initialized if `BSF_DEBUG_MODE` is enabled.